testsuite/features/build_validation/retail/sle15sp2_kiwi_image_buildhost.feature

```diff
@@ -9,7 +9,7 @@ Feature: Bootstrap a SLES 15 SP2 Salt build host via the GUI
 
   Scenario: Bootstrap a SLES 15 SP2 build host
      Given I am authorized as "admin" with password "admin"
```

Comment:
Do we still need this here? Why not use the new scenario with the login?

Reply:
Fixed.